test/e2e/framework/node/kubelet_stats.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package node
```

Comment:
Wondering if this should be `test/e2e/framework/kubelet/stats.go` instead, what do you think?

Reply:
Sounds good

Reply:
I see `test/e2e/framework/node/stats.go` but not `test/e2e/framework/kubelet/stats.go`

Reply:
sorry, my bad